### PR TITLE
Bug 976205 - Google Play uploads just one screenshot of each type

### PR DESCRIPTION
**Pull request by AlexRukhlin**

### CODE REVIEW

File: Tasks/google-play-release/GooglePlay.ts [hunk 1]
```diff
@@ -399,7 +399,7 @@ async function updateTrack(
  * Uploads change log files if specified for all the apk version codes in the update
  * @param changelogFile
  * @param apkVersionCodes
- * @returns {*}
+ * @returns nothing
```
Comment (lkillgore) [inline Tasks/google-play-release/GooglePlay.ts]: Do we really want to say 'nothing' or should we just leave this line out all together?

Comment (AlexRukhlin) [inline Tasks/google-play-release/GooglePlay.ts]: For a different programming language (which I know better) I would write "returns void". I'm not sure if that's a correct expression for TypeScript.

Comment (AlexRukhlin) [inline Tasks/google-play-release/GooglePlay.ts]: Ah! it should be "returns Promise<void>" =%-0
Crazy language....

**Comment by madhurig** [comment]
Looks good to me

**Pull request merged**